en/docs/genai/develop/agents/memory.md

```
@@ -0,0 +1,217 @@
+- You want a non-default store (for example, MSSQL).
+- You want to override the overflow strategy.
+- You want to share one memory store across multiple agents.
+
+## Sessions and Isolation
+
+Whichever store you pick, each session ID gets its own memory. If two users hit the same agent at the same time, their conversations stay separate.
+
+```
+sessionId = "user-alice-1234"  ────►  Alice's history
+sessionId = "user-bob-5678"    ────►  Bob's history
+sessionId = "user-alice-9999"  ────►  Alice's *other* conversation
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)